metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_report.py

```diff
@@ -16,18 +17,20 @@
 logger: logging.Logger = logging.getLogger(__name__)
 
 
-class BigQuerySchemaApiPerfReport:
-    list_projects = PerfTimer()
-    list_datasets = PerfTimer()
-    get_columns_for_dataset = PerfTimer()
+@dataclass
+class BigQuerySchemaApiPerfReport(Report):
+    list_projects: PerfTimer = field(default_factory=PerfTimer)
+    list_datasets: PerfTimer = field(default_factory=PerfTimer)
+    get_columns_for_dataset: PerfTimer = field(default_factory=PerfTimer)
     get_tables_for_dataset = PerfTimer()
     list_tables = PerfTimer()
```

Comment:
```suggestion
    get_tables_for_dataset: PerfTimer = field(default_factory=PerfTimer)
    list_tables: PerfTimer = field(default_factory=PerfTimer)
```
Now, all timers are covered :) .

Reply:
@hsheth2 Can you please make this change also ?